apps/system/test/unit/accessibility_test.js

```diff
@@ -247,8 +270,9 @@ suite('system/Accessibility', function() {
       accessibility.handleEvent(getAccessFuOutput(liveRegionDetails));
       assert.isFalse(stub_playSound.called);
       assert.isTrue(stubSpeak.called);
-      assert.isTrue(stubSpeak.calledWith(liveRegionDetails.data, null,
-        liveRegionDetails.options));
+      // This is where the calledWith comparison fails
+      /*assert.isTrue(stubSpeak.calledWith(liveRegionDetails.data, 
+          hintsCallback, liveRegionDetails.options));*/
```

Comment:
Correct, this should be good
assert.deepEqual(liveRegionDetails.data, stubSpeak.args[0][0]);
assert.equal(typeof stubSpeak.args[0][1], 'function');
assert.deepEqual(liveRegionDetails.options, stubSpeak.args[0][2]);